src/concepts/linearSearch.js

```diff
@@ -0,0 +1,18 @@
+/**
```

Comment:
Hey Can you create a folder in problems called `Seaeching` and move your solution there?